client/pages/index.js

```diff
@@ -1,10 +1,94 @@
+import Image from "next/image";
+
 import Hero from "../components/Hero";
 import Layout from "../components/Layout";
 
 export default function Home() {
   return (
     <Layout>
       <Hero />
+
```

Comment:
Personally think we should refactor this to it's own component, to keep the page file nice and clean